sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -964,6 +964,23 @@ export class MessageReceiver extends LinkEntity {
         this.address,
         err
       );
+      if (typeof this._onError === "function") {
+        log.error(
+          "[%s] Unable to automatically reconnect Receiver '%s' with address '%s'.",
+          connectionId,
+          this.name,
+          this.address
+        );
+        try {
+          this._onError(err);
+        } catch (err) {
+          log.error(
+            "[%s] User-code error in error handler called after disconnect: %O",
```

Comment:
Shouldn't the message say something to effect of user-code handler execution encountered an error? The existing message seems more appropriate for the try block or maybe finally.

I think one mistake users might be prone to making is expecting the errors from the code they write in callback to bubble up to their process/app - but we'd actually be eating it away. 
(Although this was already discussed in detail previously, this comment is for the logging to be updated)